interface-elements-for-web/articles/report-designer/bind-to-data/bind-a-report-to-a-union-based-federated-data-source.md

```diff
@@ -0,0 +1,85 @@
+---
+title: Bind a Report to a Union-Based Federated Data Source
+author: Boris Zaitsev
+---
+
+# Bind a Report to Union-Based Federated Data Source
+
+This topic describes how to create a union-based federated data source that combines two tables from different data sources by appending rows from one table to another.
+
+## Create a Report and Data Sources
+
+1. [Create a new blank report](../../../articles/report-designer/report-designer-tools/report-wizard/empty-report.md).
+
+2. [Add a SQL data source](../../../articles/report-designer/bind-to-data/bind-a-report-to-a-database.md) that retrieves data from the `Customers` table of the **Northwind** database.
+
+3. [Add a JSON data source](../../../articles/report-designer/bind-to-data/bind-a-report-to-json-data.md) that retrieves supplier data from the [https://services.odata.org/v4/northwind/northwind.svc/Suppliers](https://services.odata.org/v4/northwind/northwind.svc/Suppliers) location.
+
+    ![](../../../images/eurd-web-federated-datasource-union-json-datasource.png)
+
+## Create Data Federation
+
+1. Invoke the designer [menu](../report-designer-tools/menu.md) and click **Add Data Source**.
+
+   ![](../../../images/eurd-web-data-source-menu.png) 
+
+
+1. Select **Data Federation** in the invoked [Data Source Wizard](../report-designer-tools/data-source-wizard.md) and click **Next**.
+
+    ![](../../../images/eurd-web-data-source-wizard-select-data-federation.png)
+
+1. On the next page, click the **+ (plus) button** and select **Union** to add a Union query.
+
+    ![](../../../images/eurd-web-data-federation-wizard-add-union-query.png)
+
+1. In the invoked [Query Builder](../../../articles/report-designer/report-designer-tools/query-builder.md), drag and drop the table from the SQL data source onto the design surface.
+
+    ![](../../../images/eurd-web-data-federation-union-query-builder-drop-table.png)
+
+1. Drag and drop the JSON data source onto the design surface.
+
+    ![](../../../images/eurd-web-data-federation-union-query-builder-drop-json-table.png)
+
+1. The query includes only fields that have identical names and types in the original sources. You can specify a different name for a field. Click the field's **Alias** cell and enter the  new field name, if necessary. 
+
+    You can select the **Union All** checkbox in the bottom left to not remove duplicated entries from the query result.
```

Comment:
Do you mean the following?

`You can select the **Union All** checkbox in the bottom left to prevent the removal of duplicate entries from the query result.`

Reply:
```suggestion
    You can select the **Union All** checkbox in the bottom left to prevent the removal of duplicate entries from the query result.
```